sdk/core/core-http/test/policies/throttlingRetryPolicyTests.ts

```diff
@@ -114,6 +114,28 @@ describe("ThrottlingRetryPolicy", () => {
       assert.deepEqual(response, mockResponse);
     });
 
+    it("should pass the response to the handler if the status code equals 503", async () => {
+      const request = new WebResource();
+      const mockResponse = {
+        status: 503,
+        headers: new HttpHeaders({
+          "Retry-After": "100"
```

Comment:
can we assert that it delays for the appropriate amount of time while mocking the clock?